pandas/core/dtypes/common.py

```diff
@@ -120,45 +120,52 @@ def is_object_dtype(arr_or_dtype):
 
 
 def is_sparse(arr):
-    """Check whether an array-like is a pandas sparse array.
+    """
+    Check whether an array-like is a 1-D pandas sparse array.
 
     Check that the one-dimensional array-like is a pandas sparse array.
     Returns True if it is a pandas sparse array, not another type of
     sparse array.
 
     Parameters
     ----------
-    arr : array-like (1-D)
+    arr : array-like
         Array-like to check.
 
     Returns
     -------
-    boolean
+    bool
         Whether or not the array-like is a pandas sparse array.
 
     See Also
     --------
     DataFrame.to_sparse : Convert DataFrame to a SparseDataFrame.
     Series.to_sparse : Convert Series to SparseSeries.
+    Series.to_dense : Return dense representation of a Series.
```

Comment:
Based on my comment above, I think we want to include in the see also the mentioned methods `pandas.SparseArray` and `pandas.SparseSeries`. Also, may be it makes sense to get rid of `DataFrame.to_sparse`, as it's not so directly related.